Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM fedora:20
+FROM centos:7
 MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 
-# Execute system update
-RUN yum -y update && yum clean all
```

Comment:
I've added this in to the same line as the yum install stuff - one less layer that's unnecessary.